### PR TITLE
Add configuration snippet to project, projectmanager, require-matchin…

### PR DESCRIPTION
**Pull request by matthyx**
…g-label, requriesig and retitle
also normalize log message with `Warnf`
relates to #9079

### CODE REVIEW

File: prow/plugins/blockade/blockade.go [hunk 1]
```diff
@@ -74,25 +74,9 @@ func helpProvider(config *plugins.Configuration, enabledRepos []config.OrgRepo)
...
-				BlockRegexps: []string{
-					"^examples*",
-				},
-				ExceptionRegexps: []string{
-					"^examples-keep/",
-				},
-				Explanation: "examples have moved elsewhere",
-			},
-		},
-	})
```
Comment (petr-muller) [inline prow/plugins/blockade/blockade.go:93]: Why is this removed? The description does not mention it.

Comment (matthyx) [reply prow/plugins/blockade/blockade.go:93]: Good catch... I did wrong with my rebase.

**Review by petr-muller — APPROVED** [review]

**Review by cjwagner — APPROVED** [review]
